micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/RoleLanding/RoleLandingPage.js

```
@@ -1,70 +1,77 @@
 const RoleLandingComponent = () => {
-    const { t } = useTranslation();
-    const history = useHistory();
+  const { t } = useTranslation();
+  const history = useHistory();
 
-    const handleButtonClick = (action) => {
-        const url= '/'+window.contextPath+action
-        window.open(url, "_blank");
-    };
+  const handleButtonClick = (action) => {
+    const url = '/' + window.contextPath + action;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)